Taarafo.Core/Services/Foundations/Profiles/ProfileService.Validations.cs

```diff
@@ -39,6 +39,15 @@ private void ValidateProfileIsNotNull(Profile profile)
                 throw new NullProfileException();
             }
         }
+        private void ValidateStorageProfile(Profile maybeProfile, Guid profileId)
```

Comment:
Maybe, this method needs empty lines at the beginnig and the end

Reply:
done